server/src/main/java/keywhiz/service/config/Readwrite.java

```diff
@@ -0,0 +1,9 @@
+package keywhiz.service.config;
+
+import com.google.inject.BindingAnnotation;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@BindingAnnotation
+@Retention(RetentionPolicy.RUNTIME)
+public @interface Readwrite {}
```

Comment:
We already have an annotation for read-only bindings, adding a read-write annotation helps simplify code in that we can directly inject a read-write instance rather than injecting a factory and invoking the readwrite() method on it.